File: 15.py
n = int(input())
if n>=1 and n<=50:
  b = input()
  removed = 0
  c = list(b)
  for i in range(len(c)-1):
    if c[i]==c[i+1]:
      if c[i+1] not in c:
        break
      removed +=1

  print(removed)

File: 15.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>
int main(){
  int n;
  char *arr;
  int removed=0;
  scanf("%d",&n);
  if(n<=50 && n>=1){    
    arr = (char*) malloc(n * sizeof(char));
  }
  scanf("%s",arr);
  for (int i = 0; i < n; i++)
  {
    if (arr[i]==arr[i+1]){
      removed++;
    }
     
  }
  printf("%d\n",removed);
  
  
  return 0;
}